tests/jax/common.libsonnet

```diff
@@ -142,25 +206,6 @@ local tpus = import 'templates/tpus.libsonnet';
     tpuSettings+: {
       softwareVersion: 'tpu-vm-base',
```

Comment:
Do you need to override the `create-tpu` script? Shouldn't this just be a one-line to the `softwareVersion`?

Reply:
The image is not public yet.